client/src/pages/App.tsx

```diff
@@ -34,6 +34,17 @@ const GQL_GET_APP_DATA = gql`
       status
       pendingVerification
       code
+      preferences {
+        uuid
+        preference {
+          uuid
+          name
+        }
+        person {
+          uuid
+        }
```

Comment:
You probably don't need this (you already know the person linked to these preferences, as it is the parent object).